README.rst

```diff
@@ -1,52 +1,119 @@
+
+  - Windows
 
-The AFDKO requires `Python <http://www.python.org/download/>`__ 2.7.x. It does not yet support Python 3.x.
+     .. code:: sh
 
-The package is listed in the Python Package Index (PyPI), so you can
-install it with `pip <https://pip.pypa.io>`__:
+       source afdko_env/bin/activate.bat
+
+
+  Install `afdko`_:
+   .. code:: sh
+
+     pip install afdko
+
+
+  Installing the afdko inside a virtual environment prevents conflicts between its dependencies and other modules installed globally.
+
+
+**- Option 2**
+
+  Install `afdko`_ globally:
+   .. code:: sh
+
+     pip install --user afdko
+
+
+Updating
+~~~~~~~~
+Use the ``-U`` (or ``--upgrade``) option to update the afdko and its dependencies to the newest available release:
 
 .. code:: sh
 
-    pip install afdko
+    pip install -U afdko
 
-The current PyPi package is a beta release for Mac OS X.
 
-You can remove the afdko package with the command:
+Uninstalling
+~~~~~~~~~~~~
+To remove the afdko package use the command:
 
 .. code:: sh
 
     pip uninstall afdko
 
-If you have both the FDK from the Adobe AFDKO web page installed, and and the new afdko package installed, the commands in the new afdko will take precedence over commands in the older Adobe FDK, as the Python package directory is added at the beginning of the PATH directory list, and the old installer added the Adobe FDK directory to the end of the list.
 
-Note that the PyPi installer will add the new adko package paths to the start of your system PATH environment variable, and this is not undone by the uninstaller. If you want to completely clean up, you will need to change the PATH environment variable to remove the new afdko executable directories. On the Mac, this means editing the line in your login file that sets the PATH variable. On Windows, this means editing the PATH environment variable in the System control panel.
+Comments
+~~~~~~~~
+If you have both the FDK from the Adobe AFDKO web page installed, and the new afdko package installed, the commands in the new afdko will take precedence over commands in the older Adobe FDK, as the Python package directory is added at the beginning of the PATH directory list, and the old installer added the Adobe FDK directory to the end of the list.
+
+Note that the PyPI installer will add the new adko package paths to the start of your system PATH environment variable, and this is not undone by the uninstaller. If you want to completely clean up, you will need to change the PATH environment variable to remove the new afdko executable directories. On the Mac, this means editing the line in your login file that sets the PATH variable. On Windows, this means editing the PATH environment variable in the System control panel.
 
-You can also download and install older versions of the AFDKO from the `Adobe afdko home page <http://www.adobe.com/devnet/opentype/afdko.html>`__.
+You can download older versions of the tools from the `Adobe AFDKO homepage`_.
```

Comment:
Those downloads include `IS` and the old `checkOutlines`, correct? If so, I would add a note about that here.